cmd/cephcsi.go

```diff
@@ -166,6 +168,13 @@ func main() {
 		}
 	}
 
+	// read and formulate conf.Domains from passed in domain labels
+	// TODO: decide if this will be from an init container and move as a separate image or instance
+	conf.Domains, err = util.GetDomainsFromLabels(conf.DomainLabels, conf.NodeID)
```

Comment:
Also how  are we going to deal with dynamically changed domain lables?

Reply:
domain labels are set for a node, which indicate its relative location within the defined cluster domains (say rack/room/DC etc.). These are not going to change dynamically, unless the node instance is moved physically (or virtually) to another domain, at which point, I would assume, the node is restarted with the new labels, allowing us to read the new values and advertise the same.